DTW/Standard/Ingrained/map.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0"?>
+<map proto="1.4.0">
+<name>Ingrained</name>
+<version>1.0.0</version>
+<objective>Break the wool from the enemy team's monument while defending yours as well.</objective>
+<authors>
+    <author uuid="571c1d9d-4d49-43bd-bfdb-30563e920ca9">Wesdial</author>
+</authors>
+<contributors>
+ <contributor uuid="4f0ac4d7-4776-4400-9283-6bf4a5bcd48f" contribution="Team Spawn"/> <!-- Virus53 -->
+ <contributor uuid="3fa22f5e-098d-49f7-b9ad-92566f045cbc" contribution="House"/> <!-- BigTonyMC -->
+</contributors>
+<!-- Teams -->
+<teams>
+ <team id="blue-team" color="blue" max="7">Blue</team>
+ <team id="red-team" color="dark red" max="7">Red</team>
+</teams>
+<!-- Kits -->
+<kits>
+    <kit id="default">
+        <item slot="0" unbreakable="true">iron sword</item>
+        <item slot="1" unbreakable="true">bow</item>
+        <item slot="2" unbreakable="true">stone pickaxe</item>
+        <item slot="3" unbreakable="true">stone axe</item>
+        <item slot="4" unbreakable="true">stone spade</item>
+	<item slot="5" amount="64">wood</item>
+	<item slot="6" amount="64">wood</item>
+        <item slot="8" amount="64">cooked beef</item>
+        <item slot="9" amount="32">arrow</item>
+        <chestplate unbreakable="true">chainmail chestplate</chestplate>
+        <boots unbreakable="true">iron boots</boots>
+    </kit>
+    <kit id="blue-kit" parents="default">
+        <helmet unbreakable="true" color="3C44AA">leather helmet</helmet>
+        <leggings unbreakable="true" color="3C44AA">leather leggings</leggings>
+    </kit>
+    <kit id="red-kit" parents="default">
+        <helmet unbreakable="true" color="B02E26">leather helmet</helmet>
+	<leggings unbreakable="true" color="B02E26">leather leggings</leggings>
+    </kit>
+</kits>
+<!-- Spawns -->
+<spawns>
+    <spawn team="blue-team" kit="blue-kit" yaw="0.5" region="blue-spawn"/>
+    <spawn team="red-team" kit="red-kit" yaw="-180" region="red-spawn"/>
+    <default region="obs-spawn"/>
+</spawns>
+<!-- Monuments -->
+<destroyables completion="32%" repairable="false" show-progress="true">
```

Comment:
You could move `sparks="true"` and `materials="wool` from line 50-51 to line 49 inside the <destroyables> tag if you want to avoid repeating it over again.